src/application.py

```diff
@@ -26,13 +26,14 @@
 ]
 
 
-class GraphsApplication(Graphs.Application):
+class PythonApplication(Graphs.Application):
```

Comment:
I am not 100% convinced of renaming this main class, but I have no super strong opinion either. The original class name is what is set by GNOME Builder when creating a new project, so I always assumed that is the most conventional way to do this, and it's close to what most projects I see seem to be doing for the main class name.

But it's not directly a blocker for me.

Reply:
The reason here was, that the gtypename should resemble the classname with the namespace prefixed. The classname should'nt include the namespace "Graphs" itself. The problem here was that the typename "GraphsApplication" was taken by our own libgraphs (vala code). Since this was an actual class not an interface like e. g. Data I had to come up with something. I'm open for suggestions here. One idea I had was to revert to main.py as a single entry point and handle all the font registering, config migration etc. outside any class's scope. This way we do not need to inherit from our vala defined main class anymore and this problem would'nt persist.